vividus/src/main/java/org/vividus/expression/UrlExpressionProcessors.java

```diff
@@ -0,0 +1,47 @@
+package org.vividus.expression;
+
+import java.util.List;
+
+import javax.inject.Named;
+
+import org.apache.commons.lang3.Validate;
+import org.jbehave.core.expressions.DelegatingExpressionProcessor;
+import org.jbehave.core.expressions.SingleArgExpressionProcessor;
+import org.vividus.util.UriUtils;
+
+@Named
+public class UrlExpressionProcessors extends DelegatingExpressionProcessor {
+
+    private static final String PATH_AND_QUERY = "%s?%s";
+
+    public UrlExpressionProcessors() {
+        super(List.of(
+                new SingleArgExpressionProcessor<>("urlHost", UrlExpressionProcessors::getUrlHost),
+                new SingleArgExpressionProcessor<>("urlPath", UrlExpressionProcessors::getUrlPath),
+                new SingleArgExpressionProcessor<>("relativeUrl", UrlExpressionProcessors::getRelativeUrl),
```

Comment:
Do we need raltiveUrl expression? In already existing steps relativeUrl check actually checks path equality. (In PageSteps.java "Then the page has the relative URL '$relativeURL'" as example)